about.html

```diff
@@ -12,53 +12,86 @@
 </head>
 
 <body>
+
+<header>
+    <nav>
+       
+        <a href ="#"> <img class ="logo" src ="img/lambda-black.png"> </a>
+        <a href ="index.html">Home</a>
+        <a href ="about.html">About</a>
+        <a href ="#">Products</a>
+        <a href ="#">Blog</a>
+        <a href ="#">Contact</a> 
+    </nav>
+</header>
+
+<img class ="my-top-image" src ="img/jumbo-about.png">
+
     <div class="container about-page">
-     
-    About
+     <article class ="top-text">
+       <h1> About</h1>
+       Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+     </article>
+    </div>
+<hr>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+<div class ="cont-parent">
 
+     <div class ="container-child-one">
 
-    <img src="img/about-plan.png" alt="strategy">
+        <img src="img/about-plan.png" alt="strategy">
+  
+        <p>Strategy</p>
     
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+       Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.<br>
 
+       <button>Learn More</button>
 
+</div>
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
+     <div class ="container-child-two">
+         <img src="img/about-working.png" alt="strategy">
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        <p>How We Work</p> 
 
-    Learn More
+      Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.<br>
+      
+         <button>Learn More</button> 
```

Comment:
Think about which CSS properties may allow you to style the button to more closely match the design file.